src/modules/commentsModal.js

```diff
@@ -80,24 +84,46 @@ const generateModal = (id) => {
     h2Comments.appendChild(h2CommentsText);
     h2AddComment.appendChild(h2AddCommentText);
     submitBtn.appendChild(submitBtnText);
+  });
+
+  // Appends elements to the modal
+  commentsModal.appendChild(closeBtn);
+  commentsModal.appendChild(img);
+  commentsModal.appendChild(title);
+  commentsModal.appendChild(summary);
+  commentsModal.appendChild(ulDetails);
+  commentsModal.appendChild(h2Comments);
+  commentsModal.appendChild(ulComments);
+  commentsModal.appendChild(h2AddComment);
+  commentsModal.appendChild(form);
+  form.appendChild(nameInput);
+  form.appendChild(commentTextArea);
+  form.appendChild(submitBtn);
```

Comment:
nice work my friend I really like how do you create the components using document.createElement!
also the way that you create first all the components then add the classes! how do you feel it?


I approve the review, merged and deleted the new feature
just if you want you can improve this

instead of call a lot of times commetsModal.appendChild
you can do it with ***append***

`commentsModal.append(closeBtn, img, title, summary, ulDetails, h2Comments, ulComments, h2AddComment, form)`